src/settings-ui/Settings.UI/Views/PowerPreviewPage.xaml

```diff
@@ -78,6 +78,16 @@
                                     IsChecked="{x:Bind ViewModel.MonacoPreviewTryFormat, Mode=TwoWay}"
                                     IsEnabled="{x:Bind ViewModel.MonacoRenderIsEnabled, Mode=OneWay}" />
                             </labs:SettingsCard>
+                            <labs:SettingsCard
+                                x:Uid="FileExplorerPreview_Toggle_Monaco_Max_File_Size"
+                                IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.MonacoRenderIsEnabled}">
+                                <NumberBox
+                                    MinWidth="{StaticResource SettingActionControlMinWidth}"
+                                    Maximum="100000"
+                                    Minimum="2000"
```

Comment:
Any statistics for these numbers? Just curious.

Reply:
No. But anything outside these values is nonsense. And if people need more they will tell us this for sure.